test/integration/storage.integration.test.js

```diff
@@ -81,9 +116,41 @@ describe.only("uploadDocumentAtId", () => {
     const uploaded = uploadDocumentAtId(document, queueNumber);
     await expect(uploaded).rejects.toThrow("Document is not valid");
   });
+
+  it("should allow user to specify ttl", async () => {
+    isValid.mockReturnValueOnce(true);
+    const { id: queueNumber } = await getQueueNumber();
+    const document = { foo: "bar" };
+    const uploaded = await uploadDocumentAtId(document, queueNumber, 20000);
+    expect(uploaded).toMatchObject(thatIsUploadResponse);
+    const getResults = await getDocument(uploaded.id);
+    expect(
+      getResults.document.ttl < Date.now() + 20000 + TIME_SKEW_ALLOWANCE
```

Comment:
jest has matchers for numerical operations
https://jestjs.io/docs/en/using-matchers#numbers